packages/database/src/core/RepoManager.ts

```diff
@@ -108,16 +113,30 @@ export class RepoManager {
     let parsedUrl = parseRepoInfo(dbUrl);
     let repoInfo = parsedUrl.repoInfo;
 
+    let authTokenProvider: AuthTokenProvider;
+
+    let isEmulator: boolean;
+
     let dbEmulatorHost: string | undefined = undefined;
     if (typeof process !== 'undefined') {
       dbEmulatorHost = process.env[FIREBASE_DATABASE_EMULATOR_HOST_VAR];
     }
+
     if (dbEmulatorHost) {
+      isEmulator = true;
       dbUrl = `http://${dbEmulatorHost}?ns=${repoInfo.namespace}`;
       parsedUrl = parseRepoInfo(dbUrl);
       repoInfo = parsedUrl.repoInfo;
+    } else {
+      isEmulator =
+        parsedUrl.repoInfo.host === 'localhost' && !parsedUrl.repoInfo.secure;
```

Comment:
We do see people running their emulator on non-localhost hosts.  Like `0.0.0.0`.  I think just anything non-secure (`http://`) is the right condition.

Reply:
Done. I will verify this manually on Monday, which should allow us to release this next week.